### PR TITLE
[master] DG-547 Ignore cached entity & fetch latest

### PR DESCRIPTION
**Pull request by nikhilbonte21**

### CODE REVIEW

**Review by n5nk — APPROVED** [review]

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java [hunk 1]
```diff
@@ -3297,7 +3305,7 @@ public void updateClassifications(EntityMutationContext context, String guid, Li
             String      entityGuid = graphHelper.getGuid(vertex);
             AtlasEntity entity     = instanceConverter.getAndCacheEntity(entityGuid, ENTITY_CHANGE_NOTIFY_IGNORE_RELATIONSHIP_ATTRIBUTES);
 
-            if (isActive(entity)) {
```
Comment (sumandas0) [inline repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java:3300]: Why this change?

Comment (sumandas0) [inline repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java:3300]: Why this change?

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java [hunk 2]
```diff
@@ -160,6 +187,53 @@ public void setClassifications(Map<String, AtlasEntityHeader> map) throws AtlasB
...
+
+                        allVertices.add(vertex);
+                        propagatedEntities.add(entity);
+                    }
+
+                    entityChangeNotifier.onClassificationsAddedToEntities(propagatedEntities, Collections.singletonList(addedClassification), false);
+                }
+            }
+            entityGraphMapper.updateClassificationText(null, allVertices);
+            transactionInterceptHelper.intercept();
```
Comment (sumandas0) [inline repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java:233]: We will be commiting updated classifications in `entitiesStore.updateClassifications(entityGuid, list);`,

Can updating this part later will break the atomicity of the request or it will be committed at last?

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java (hunk 1 above)
Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java:3300]: Cause if we are adding the classification we will always keep it in the classiifcation text?

Comment (nikhilbonte21) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java:3300]: Skipping archived entities again makes __classificationText inconsistent

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java (hunk 2 above)
Comment (nikhilbonte21) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java:233]: It will be comited at last once for all assets there in request, also it will only commit __classificationText as other writes are already comitted in add/update or delete methods

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java (hunk 1 above)
Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java:3300]: So we want to discover archived entity if we have tag attached? right

File: repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java (hunk 2 above)
Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/store/graph/v2/ClassificationAssociator.java:233]: So it is two step write? First all the assets then at last we will commit classification text

**Review by sumandas0 — APPROVED** [review]
